docs/tutorials/aws-sd.md

```diff
@@ -19,14 +19,28 @@ To use the AWS Cloud Map API, a user must have permissions to create the DNS nam
 {
   "Version": "2012-10-17",
   "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "route53:ChangeResourceRecordSets"
+      ],
+      "Resource": [
+        "arn:aws:route53:::hostedzone/*"
+      ],
+      "Condition": {
+        "StringEquals": {
+          "aws:SourceVpc": ["vpc-00000000"]
+        },
+        "ForAllValues:StringLike": {
+          "route53:ChangeResourceRecordSetsNormalizedRecordNames": "*example.com"
+        }
+      }
+    },
```

Comment:
It's a good example. Mmmh :thinking:, at the same time, the current doc structure is interesting: 

1. first example: required simple permission, ready to use
2. second example (L60+ in this PR): advanced example, with ABAC

=> Wdyt of adding this condition example in the second section ?

Reply:
Make sense. Changed, added comments